### PR TITLE
Doc: Add instructions in PR template to choose SIGs

### LINKED ISSUE #97725: Add instructions in PR template to choose SIGs
**What would you like to be added**:

Add instructions in PR template to choose SIGs.

**Why is this needed**:

Now we need both Kind and SIG labels before a PR can be merged(refer to kubernetes/test-infra/pull/19827). Even though fejta-bot can help us identify SIGs in some cases, but  sometimes it is difficult for bot to add the right and full SIGs involved. As we can see, [PRs with do-not-merge/needs-sig label]( https://github.com/kubernetes/kubernetes/pulls?q=is%3Apr+is%3Aopen+label%3Ado-not-merge%2Fneeds-sig) still include some new created PRs. We can add a section to instruct contributors to add SIG labels when they create a PR just like Kind labels.

/sig contributor-experience

### PR DESCRIPTION
**Pull request by gavinfish**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Add instructions in PR template to choose SIGs.

**Which issue(s) this PR fixes**:

Fixes #97725

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

/sig contributor-experience

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -26,6 +26,36 @@ Optionally add one or more of the following kinds if applicable:
 /kind regression
 -->
 
+**Which SIG(s) this PR belongs to?**
+
+<!--
+Add one or more of the following SIGs:
```
Comment (guineveresaenger) [inline .github/PULL_REQUEST_TEMPLATE.md:32]: Please add a link to the contributor documentation that explains SIGs and how to find the proper SIG!

**Review by guineveresaenger — APPROVED** [review]
I like this change. Thank you!

**Comment by guineveresaenger** [comment]
@gavinfish thank you for this PR! 🎉 

I'm sorry this has not received attention in so long.

Could you please resolve the merge conflict and retest?

/lgtm

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (gavinfish) [reply .github/PULL_REQUEST_TEMPLATE.md:32]: Done.

**Comment by guineveresaenger** [comment]
/lgtm

paging @mrbobbytables to /approve

**Review by mrbobbytables — COMMENTED** [review]
I don't want to downplay the idea of a comment about adding a SIG, but I think the real solution is to get the appropriate OWNERS files updated to include the label. SIGs rarely triage PRs that don't have a SIG assigned already. We've invested in a lot to ensure we do have OWNER coverage and the fact that we have a few areas that don't seems like an oversight we REALLY should resolve.

This also adds a very large chunk to an already cluttered PR template =/ 
I'd be okay with adding  a 1 line comment like
```
<!-- OPTIONAL: Add additional SIGs with  /sig <foo> -->
```
or something similar, but I really think we should resolve the label oversight as that seems to be the root cause of the problem.

**Comment by nikhita** [comment]
> I think the real solution is to get the appropriate OWNERS files updated to include the label. 

:+1:

**Comment by gavinfish** [comment]
> I don't want to downplay the idea of a comment about adding a SIG, but I think the real solution is to get the appropriate OWNERS files updated to include the label. SIGs rarely triage PRs that don't have a SIG assigned already. We've invested in a lot to ensure we do have OWNER coverage and the fact that we have a few areas that don't seems like an oversight we REALLY should resolve.
> 
> This also adds a very large chunk to an already cluttered PR template =/
> I'd be okay with adding a 1 line comment like
> 
> ```
> 
> ```
> 
> or something similar, but I really think we should resolve the label oversight as that seems to be the root cause of the problem.

Make sense. We should depend on bot to detect related SIGs. Only for few cases like feature flag etc, it may be hard to automatically set the SIG label. A simple notification to let contributor consider SIG is good to me.

**Pull request closed without merging**